.jenkinsci/debug-build.groovy

```diff
@@ -23,6 +23,10 @@ def doDebugBuild(coverageEnabled=false) {
     parallelism = 1
   }
 
+  // for saving coredumps
+  sh "echo %e.%p.coredump > /proc/sys/kernel/core_pattern"
```

Comment:
Can we make coredumps collection option configurable? That can be disabled by request (using Jeknins GUI). It should be enabled by default